tests/functional/cypress/e2e/form-design-basicfields.cy.js

```diff
@@ -235,20 +235,25 @@ describe('Form Designer', () => {
       cy.get('button').contains('Basic Fields').click();
       cy.get('div.formio-builder-form').then($el => {
         const coords = $el[0].getBoundingClientRect();
-        cy.get('span.btn').contains('Text Field')
+        cy.get('span.btn').contains('Checkbox')
         
         .trigger('mousedown', { which: 1}, { force: true })
         .trigger('mousemove', coords.x, -50, { force: true })
         .trigger('mouseup', { force: true });
         
         cy.get('button').contains('Save').click();
       });
-      //cy.get('[ref=removeComponent]').then($el => {
 
-       // const rem=$el[11];
-        //rem.click();
+      cy.waitForLoad();
+      cy.waitForLoad();
```

Comment:
do we need two waitforLoads here ?

Reply:
yes, sometimes cypress needs time to locate elements

Reply:
@abhilash-aot I changed to cypress wait function rather than calling wait loader function multiple times